code/modules/modular_computers/computers/item/computer.dm

```diff
@@ -31,6 +31,7 @@
 	var/steel_sheet_cost = 5								// Amount of steel sheets refunded when disassembling an empty frame of this computer.
 
 	obj_integrity = 100
+	integrity_failure = 50
```

Comment:
Tthe item doesn't have a version of obj_break(), so why give it a failure level?